src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -179,27 +179,82 @@ const saveAndContinue = async () => {
   await flushPromiseQueue()
 }
 
+const recommendedAddress = {
+  addressLine1: "401 Broadway Suite 25",
+  addressLine2: null,
+  city: "New York",
+  region: "NY",
+  postalCode: "10013",
+  country: "US",
+}
+
+const verifyAddressWithSuggestions = async (relayEnv, input, suggested) => {
+  const addressLines = addr => {
+    return [
+      addr.addressLine1,
+      addr.addressLine2,
+      `${addr.city}, ${addr.region} ${addr.postalCode}`,
+      addr.country,
+    ]
+  }
+
+  const verificationResult = {
+    __typename: "VerifyAddressType",
+    verificationStatus: "VERIFIED_WITH_CHANGES",
+    suggestedAddresses: [
+      {
+        lines: addressLines(recommendedAddress),
+        address: recommendedAddress,
+      },
+    ],
+    inputAddress: {
+      lines: addressLines(validAddress),
+      address: validAddress,
+    },
+  }
+
+  const mutation = relayEnv.mock.getMostRecentOperation()
+  expect(mutation.request.node.operation.name).toEqual(
+    "AddressVerificationFlowQuery"
+  )
+  expect(mutation.request.variables).toEqual({
+    address: {
+      addressLine1: input.addressLine1,
+      addressLine2: input.addressLine2,
+      city: input.city,
+      region: input.region,
+      postalCode: input.postalCode,
+      country: input.country,
+    },
+  })
+
+  relayEnv.mock.resolveMostRecentOperation(operation => {
+    return MockPayloadGenerator.generate(operation, {
+      VerifyAddressType: () => verificationResult,
+    })
+  })
+  expect(await screen.findByText("Confirm your delivery address")).toBeVisible()
+  expect(await screen.findByText("Recommended")).toBeVisible()
+  expect(await screen.findByText("What you entered")).toBeVisible()
+}
+
 describe("Shipping", () => {
   const pushMock = jest.fn()
   let isCommittingMutation
   let relayEnv
 
   beforeEach(() => {
     isCommittingMutation = false
-  })
-
-  afterEach(() => {
-    jest.clearAllMocks()
-    mockCommitMutation.mockReset()
-  })
-
-  beforeAll(() => {
     ;(useTracking as jest.Mock).mockImplementation(() => ({
       trackEvent: jest.fn(),
     }))
     ;(useFeatureFlag as jest.Mock).mockImplementation(() => false)
   })
 
+  afterEach(() => {
+    jest.restoreAllMocks()
```

Comment:
Unrelated changes to this PR. It's a [followup]([891b251](https://github.com/artsy/force/pull/12939/commits/891b2519762ca77dcc624c54f9bc0c02280ea9d0)) from https://github.com/artsy/force/pull/12929 and I learned `restore` is the right way to do.